documentation/asciidoc/topics/ref_label_environment_variables.adoc

```diff
@@ -1,6 +1,6 @@
 [id='label_environment_variables-{context}']
 = Global Labels for {ispn_operator}
-Global labels are automatically propagated to all {brandname} resources.
+Global labels are automatically propagated to all {brandname} pods.
```

Comment:
```suggestion
Global labels are automatically propagated to all {brandname} pods and services.
```

Reply:
@rigazilla Thanks, this relates to my other question about the svc labels. I changed "resources" to "pods" because it felt like "resources" was too vague. 

If we say "pods and services" will that make sense downstream?

Reply:
I think it makes sense if we add the comment below about `INFINISPAN_OPERATOR_TARGET_LABELS`